bee-queue/arena
# Promote delayed job in bull

This PR is for adding a button to promote delayed jobs for bull. Also I added an example for bullmq (note: this is just a simple example).
In order to test this new feature please:
1. Go into example directory.
2. run: npm install
3. run: npm run start:bullmq
4. Open arena in http://localhost:4735/Queue%20Server%201/name_of_my_queue

![Selection_019](https://user-images.githubusercontent.com/14021523/113449262-11b65580-93c3-11eb-8143-80a1cda4820f.png)

5. Go to delayed jobs
6. Click in the job row
7. Click in Promote button

![Selection_020](https://user-images.githubusercontent.com/14021523/113449272-167b0980-93c3-11eb-9cb3-4ed89704ff50.png)

8. You now should see that the job is in waiting 

![Selection_021](https://user-images.githubusercontent.com/14021523/113449283-1a0e9080-93c3-11eb-93a9-d02b8876bc6d.png)
